Automattic/jetpack
# Move Forms blocks to the Forms package

Part of #28405

## Proposed changes:

* Moves Form blocks from `plugins/jetpack/extensions/blocks/contact-form` into `packages/forms/src/blocks` 
* Adds necessary tooling for building the blocks
* Loads the editor scripts whenever necessary

### Other information:


## Jetpack product discussion
pcsBup-N3-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the FOrms package on your jetpack install by making sure the filter below is added before the contact form module is loaded:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
```

Make sure the package is installed with `jetpack install packages/forms && jetpack install plugins/jetpack`, and built the JS bundle with `jetpack build packages/forms`. You can also test the watch functionality with `jetpack watch packages/forms`.

Now, check if the Form blocks still work in the editor as they have worked before
